README.md

```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
 
 ## Help and Support
 
-If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Also make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
+If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
 
-It is also recommended to check the wiki first, it may have your fix:
-[Common Issues](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Common-Issues-FAQ)
+It is also recommended checking the wiki first, it may have your fix:
+[Common Issues][Common Issues].
 
 ## Bugs
 
-Please report any bugs you find [here][issues]. Try to be as descriptive as possible and make sure to check if the bug has already been reported _before_ reporting it youself. Please only report one bug per issue - if you have encountered multiple issues, report multiple bugs as seperate    issues.
+Please report any bugs you find [here][issues]. Try to be as descriptive as possible and make sure to check if the bug has already been reported _before_ reporting it yourself. Please only report one bug per issue - if you have encountered multiple issues, report multiple bugs as separate issues.
 
 ## Feature Requests
 
-New features should also be reported as [issues]. Please try to describe the feature you are suggesting in as much detail as possible and explain _why_ it would be useful. As explained above, please first check the feature hasn't already been requested and make sure your issue only covers one feature. If you have multiple ideas, open seperate issues.
+New features should also be reported as [issues]. Please try to describe the feature you are suggesting in as much detail as possible and explain _why_ it would be useful. As explained above, please first check the feature hasn't already been requested and make sure your issue only covers one feature. If you have multiple ideas, open separate issues.
 
 ## Credit
 
-   Impact is developed by [ZeroMemes], [leijurv] and [LeafHacker]. It is built on top of [ClientAPI] and supports [Baritone] which is developed by [leijurv]
+Impact is developed by [ZeroMemes], [leijurv] and [LeafHacker]. It is built on top of [ClientAPI] and supports [Baritone], which is developed by [leijurv].
```

Comment:
This was probably intended for a reason

Reply:
stay mad

Reply:
not that i can see so i think it's fine idk